tox.ini

```diff
@@ -0,0 +1,27 @@
+[tox]
+envlist = py27,py34,py35,flake8,cov
+
+[testenv]
+commands = py.test {posargs}
+basepython =
+    py27: python2.7
+    py34: python3.4
+    py35: python3.5
+    flake8: python2.7
```

Comment:
I just realized I don't have to specify the interpreters for environments named `pyXY`: https://testrun.org/tox/latest/example/general.html#basepython-defaults-overriding

So do we want the flake8 env bound to 3.5 or just run with the python version that tox is run with?